app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitors: dict) -> str:
+        if "vaccine" not in visitors.keys():
+            raise NotVaccinatedError
+        if visitors["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
```

Comment:
The `OutdatedVaccineError` should include a descriptive message when raised. According to the checklist, use descriptive error messages: `raise OutdatedVaccineError("Visitor should update their vaccine!")`.